lib/metro-transit.js

```diff
@@ -16,23 +16,21 @@ var MetroTransit = function () {
 MetroTransit.prototype = Object.create(AlexaSkill.prototype);
 MetroTransit.prototype.constructor = MetroTransit;
 
-MetroTransit.prototype.eventHandlers = {
-  onSessionStarted: function (sessionStartedRequest, session) {
-    console.log('MetroTransit onSessionStarted requestId: ' + sessionStartedRequest.requestId +
-        ', sessionId: ' + session.sessionId);
-    // any initialization logic goes here
-  },
+MetroTransit.prototype.eventHandlers.onSessionStarted = function (sessionStartedRequest, session) {
```

Comment:
The difference is very clear in the debugger. Now, I want to understand why setting the value of `MetroTransit.prototype.eventHandlers` sets it where you would assume it sets it, but setting `MetroTransit.prototype.eventHandlers.onLaunch` sets it at the `AlexaSkill.prototype.eventHandlers.onLaunch` level.